stf/model/prediction/lightgbm/lightgbm.py

```diff
@@ -0,0 +1,123 @@
+# SPDX-FileCopyrightText: 2017-2021 Alliander N.V. <korte.termijn.prognoses@alliander.com> # noqa E501>
```

Comment:
This seems almost? a 100% duplicate of `stf/model/prediction/xgboost/xgboost.py` Perhaps it makes more sense to add the methods that are used by all implementations (xgb, ligthgbm) to the abstract bass class or some utility functions.

Reply:
issue was already created;)